app/controllers/shops_controller.rb

```diff
@@ -2,6 +2,10 @@ class ShopsController < ApplicationController
   before_action :set_user_shop, only: [:edit, :update, :destroy]
   before_action :authenticate_user!, except: [:index, :show]
 
+  add_breadcrumb "Shops", :shops_path
+
+  add_breadcrumb "New Shop", :new_shop_path
```

Comment:
Yes, but it shouldn't be getting that tho. This breadcrumb for single shop page does not make sense: ` Shops > New Shop > My Cool Shop`; whereas it should be this: ` Shops > My Cool Shop`